src/popup.tsx

```diff
@@ -147,6 +161,26 @@ const Popup = () => {
             </button>
           </div>
         </form>
+        <div className="flex items-center gap-x-2">
+          {DEFAULT_COLOR.map((colorOption) => (
+            <input
+              type="radio"
+              name="color"
+              className={`bg-${colorOption}-500 w-4 h-4 rounded-full`}
+              style={{
```

Comment:
use className or style

Reply:
ok,I'll use inline styles. Initially, I used className, but it had no effect, so I switched to style. I also noticed that Chrome's tab groups support 'grey' by default, but Tailwind CSS uses the American spelling 'gray'.

Reply:
Maybe you can modify the `tailwind.config.js`